translations/translations.json

```diff
@@ -490,7 +490,10 @@
       "close": "Schließen",
       "showMore": "Mehr anzeigen",
       "showLess": "Weniger anzeigen",
-      "optional": "optional"
+      "optional": "optional",
+      "back": "Zurück",
+      "scrollLeft": "Nach links",
+      "scrollRight": "Nach rechts"
```

Comment:
:wrench: I think this is wrong on RTL layouts. Better would be something like `previous` and `next` I guess.